### ISSUE TITLE
polymer-analyzer is reporting that stylesheet imports are lazy; found this when I getFeatures all imports and inspect their lazy property.  Investigating as part of fix for https://github.com/Polymer/polymer-cli/issues/749

### ISSUE THREAD

**Issue by usergenic** [Polymer/polymer-analyzer#699]
According to https://github.com/Polymer/polymer-analyzer/blob/master/src/html/html-style-scanner.ts#L54 we mark all html stylesheet links as "lazy" imports.

What is intended by this?  Anywhere we care about the non-lazy html import graph and the related assets like css+js, I filter imports based on `!lazy` to exclude lazy-import type html imports, but maybe I have to be more specific now to only care about `filter((i) => !(type==='html-import' && lazy===true)`?

**Comment by usergenic** [comment]
```javascript
  /**
   * If true, the imported document may not be loaded until well after the
   * containing document has been evaluated, and indeed may never load.
   */
```
^ https://github.com/Polymer/polymer-analyzer/blob/master/src/model/import.ts#L54

**Comment by rictic** [comment]
Hm, this seems like a mistake to me. The stylesheet unconditionally loads when its containing document loads.
